### ISSUE TITLE
URL paths for RHEL 7.2 SDK and ASP.NET Core Runtime installer should omit “rhel” on github page - 2.1 master branch

### ISSUE THREAD

**Issue by yuki233** [dotnet/cli#8929]
URL paths for RHEL 7.2 SDK and ASP.NET Core Runtime installer should omit “rhel” on https://github.com/dotnet/versions/tree/master/build-info/dotnet/product/cli/release/2.1. Otherwise, it will return 404 error.
![image](https://user-images.githubusercontent.com/1578239/38073183-8d3b6e2c-335c-11e8-9b3a-f4e443ced12c.png)

Workaround: Modify the link ***-**rhel-x64.rpm** -> ***-**x64.rpm**, 
SDK:
https://dotnetfeed.blob.core.windows.net/orchestrated-release-2-1/20180326-03/final/assets/Sdk/2.1.300-preview2-008407/dotnet-sdk-2.1.300-preview2-008407-**rhel-x64.rpm** 
->
 https://dotnetfeed.blob.core.windows.net/orchestrated-release-2-1/20180326-03/final/assets/Sdk/2.1.300-preview2-008407/dotnet-sdk-2.1.300-preview2-008407**-x64.rpm**
 
ASP.NET Core Runtime:
https://dotnetfeed.blob.core.windows.net/orchestrated-release-2-1/20180326-03/final/assets/aspnetcore/Runtime/2.1.0-preview2-30478/aspnetcore-runtime-2.1.0-preview2-30478-**rhel.7-x64.rpm** 
 -> 
https://dotnetfeed.blob.core.windows.net/orchestrated-release-2-1/20180326-03/final/assets/aspnetcore/Runtime/2.1.0-preview2-30478/aspnetcore-runtime-2.1.0-preview2-30478-**x64.rpm**

**Comment by livarcocc** [comment]
@johnbeisner can you take a look at this?

**Comment by dagood** [comment]
@mmitche, can you help correct this?

**Comment by mmitche** [comment]
Yeah.  I thought I had removed all of the rhel bits.  Checking.

**Comment by johnbeisner** [comment]
@mmitche 
Thanks Matt...

**Comment by mmitche** [comment]
@livarcocc What's the OS support for that rpm installer?  Should I change the tag from RHEL 7.2 to something else?

**Comment by johnbeisner** [comment]
RHEL 7.2 should use a 'non-rhel' labeled RPM package: http://dotnetfeed.blob.core.windows.net/orchestrated-release-2-1/20180326-03/final/assets/Sdk/2.1.300-preview2-008407/dotnet-sdk-2.1.300-preview2-008407-x64.rpm

**Comment by mmitche** [comment]
Right, but the implication of that rpm installer losing it's 'rhel' label is that it might also be usable for other RPM based OS's? CentOS, OpenSuSE, etc.  Should the distro column be changed from RHEL7.2 to something else, like the debian based installer has?

**Comment by johnbeisner** [comment]
@mmitche 
RHEL 7.2 uses the portable Linux tarball: dotnet-sdk-2.1.300-preview2-008533-linux-x64.tar.gz
Moving forward, RHEL 7.2 will use a non-distro-specific RPM installer: dotnet-sdk-2.1.300-preview2-008533-x64.rpm

IMHO; the non-distro-specific RPM installer link should move into 'Linux x64' below the non-distro-specific DEB installer link - 'RHEL 7.2' should be removed.

**Comment by mmitche** [comment]
@johnbeisner Okay thanks.

**Comment by livarcocc** [comment]
@johnbeisner is there an expected fix here or is the issue addressed?

**Comment by wli3** [comment]
This issue was moved to dotnet/versions#290
